### DISCUSSION TITLE
Comment modifier l 'interfaces de nova 3

### DISCUSSION THREAD

**Discussion by Linxs19** [laravel/nova-issues#6534] [Q&A] [answered]
bonjour , je voudrais savoir comment modifier l'interface de laravel nova 3 en créant mes propres vue.js

**Comment by jeremynikolic** [comment, marked as answer]
Merci de nous contacter 🙏 

Vous pouvez trouver la documentation pour la version 3 de Nova ici -> [Doc v3](https://github.com/laravel/nova-docs/tree/3.x)
Et plus précisement la section sur la "Customization" ici -> [Customization](https://github.com/laravel/nova-docs/tree/3.x/customization)

Pour information nous recommandons de tenir a jour Nova avec les dernières mise à jour afin de bénéficier de l'intégralité des nouvelles fonctionnalités et potentielles corrections de bugs.
Concernant la customization, la versoin 5 de nova (publié sous peu) contiendra des améliorations quant a la création de composants et outils custom.

Si vous avez des questions plus ciblé je vous laisse ouvrir un ticket via le support Nova ou un ticket "Issues" pour les bugs si vous en rencontrer.
Ce flux de discussions étant réservé aux demandes de nouvelles fonctionnalités.
